### ISSUE TITLE
A test issue, please ignore

### ISSUE THREAD

**Issue by bob-the-hamster** [ohrrpgce/ohrrpgce#21]
this is a test issue

**Comment by bob-the-hamster** [comment]
this is a test comment on a test issue

**Comment by ohrrpgce-githubmail** [comment]
Alright! We finally have integration between the github issue tracker
https://github.com/ohrrpgce/ohrrpgce/issues/ with the mailing list

**Comment by bob-the-hamster** [comment]
closing this test bug, it has served its purpose

**Comment by rversteegen** [comment]
Eeek! Just last night before bed I noticed your comments on bug #1 a week ago, which I had missed because I wasn't actually subscribed to this repo! I was going to bring up subscribing the mailing list. So is creating a new github account and subscribing it the only way to get emails sent to a mailing list?

And I wonder, am I going to have to now unwatch this repo (with my personal GH account) to avoid duplicate emails?

I also just noticed the ability to transfer issues from one repo to another. Maaaybe I will rename this repo, create a new ohrrpgce project, import the bugzilla and sf bugs into it, then transfer all the issues from this repo to the new one. I see I will just need to manually re-apply the labels on these 21 issues. It [is possible](https://help.github.com/en/articles/renaming-a-repository) to rename a repo and replace it with another one of the same name.

**Comment by bob-the-hamster** [comment]
On Sun, Jul 28, 2019, 10:39 PM Ralph Versteegen <notifications@github.com>
wrote:

> Eeek! Just last night before bed I noticed your comments on bug #1
> <https://github.com/ohrrpgce/ohrrpgce/issues/1> a week ago, which I had
> missed because I wasn't actually subscribed to this repo! I was going to
> bring up subscribing the mailing list. So is creating a new github account
> and subscribing it the only way to get emails sent to a mailing list?
>

Yes, I created a separate GitHub account with a separate email that
forwards to the list.

And I wonder, am I going to have to now unwatch this repo (with my personal
> GH account) to avoid duplicate emails?
>

I don't think so... At any rate, I have not been getting duplicates.

I also just noticed the ability to transfer issues from one repo to

**Comment by rversteegen** [comment]
Well, I got a couple duplicate emails from this thread -- but not from every comment.

Oh, I also clicked the "mute this thread" link in on of the emails to see what would happen, and I got a message that I had muted the thread... but it isn't actually muted for me, does that apply to the ohrrpgce-githubmail account? Apparently not, because your latest comment ended up in the archives for the mailinglist, although I didn't get that email from the mailing list. I guess because it was a duplicate email that gmail filtered out? I don't know whether it filters duplicates from mailinglist.

I also notice that emails from ohrrpgce-githubmail forwarded by the mailinglist have two reply-to addresses, github and the mailinglist, which is interesting and useful.
